statusmotron_ui/lib/statusmotron_ui_web/controllers/page_controller.ex

```diff
@@ -2,6 +2,8 @@ defmodule StatusmotronUIWeb.PageController do
   use StatusmotronUIWeb, :controller
 
   def index(conn, _params) do
-    render conn, "index.html", :ets.lookup(:dumping_ground, :color)
+    conn
+    |> assign(:color, StatuslightControl.Statuslight.current_color())
+    |> render("index.html")
```

Comment:
👍  Someday I'll learn to use pipelines for stuff.  It definitely doesn't come naturally to me yet.

Reply:
❤️ me some pipeline 😂